docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  new_fastapi_db:
+    image: postgres:16
+    container_name: new_fastapi_db_container
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER:-default_user}
```

Comment:
There won't be an issue explicitly setting the value in the Docker Compose file, because we only use it locally. In production, we pass different values.

Reply:
Got it , but it's ok to leave it like this ?